core/src/main/kotlin/com/malinskiy/marathon/execution/strategy/impl/batching/IsolateBatchingStrategy.kt

```diff
@@ -7,4 +7,14 @@ import java.util.*
 
 class IsolateBatchingStrategy : BatchingStrategy {
     override fun process(queue: Queue<Test>): TestBatch = TestBatch(listOf(queue.poll()))
+
+    override fun hashCode(): Int {
```

Comment:
Redundant override

Reply:
Required by detekt

Reply:
but it broke hashcode + equals "contract" 
> If two objects are equal, then they must have the same hash code